dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
@cossou I think there is a bit of a language barrier, I'm not opting to separate the index.css file to separate files. What I want to make sure we don't do is:

Combine bootstrap.min.css + font-awesome.min.css + rdash.min.css AND THEN include index.css with it as well.

We should have
- main.min.css
- custom or index.min.css

Trust me, having our custom code among the other css files (bootstrap, font-awesome, rdash) means that in a month or a year you won't be able to fix CSS issues easily as you won't know what was fetched from components/* and what was in your custom CSS file.

I shall do a PR with the index.css file added to the minification process as a separate entity (e.g. index.min.css).